<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerModelClipModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerModelClipModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="qMRMLTreeView" name="MRMLTreeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sceneModelType">
      <string>ModelHierarchy</string>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLModelHierarchyNode</string>
       <string>vtkMRMLModelNode</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="clipCollapsibleButton">
     <property name="text">
      <string>Clip the Model</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout1">
        <item>
         <widget class="QLabel" name="clipLabel">
          <property name="text">
           <string>Select a model to clip</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="clipNodeComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLModelNode</string>
           </stringlist>
          </property>
          <property name="showChildNodeTypes">
           <bool>false</bool>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="removeEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout2">
        <item>
         <widget class="QPushButton" name="createButton">
          <property name="text">
           <string>Create a Plane</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="appendButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Append a Plane</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove the Last Plane</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout3">
        <item>
         <widget class="QPushButton" name="hideButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Hide the Planes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete the Planes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="depthButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Set Clip Depth</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="clipButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Clip</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLTreeView.h</header>
   <slots>
    <slot>setSceneModel(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerModelClipModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLTreeView</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerModelClipModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>clipNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
